client/src/pages/Game.js

```diff
@@ -31,46 +45,92 @@ const getCurrentSpymaster = ({ gameState: { turn, redSpy, blueSpy } }) => {
 }
 
 const GamePage = ({ classes, state, setState, socket }) => {
-  const { gameID } = useParams()
-  const { gameState, player } = state
+  const { gameID } = useParams();
+  const { gameState, player } = state;
+  const [timer, setTimer] = useState(20);
```

Comment:
should we accept time as a prop? could define as environment variable and propagate down, would allow for easily changing value for testing/etc

Reply:
Technically, the time is received and updated from server side. Client side should only display this value. Though, such changes can be made on the server side.